src/utils/signaling.js

```diff
@@ -356,9 +357,7 @@ Signaling.Internal.prototype._sendMessageWithCallback = function(ev) {
 		}.bind(this))
 		.catch(function(err) {
 			console.error(err)
-			showError(t('spreed', 'Sending signaling message has failed.'), {
-				timeout: 15,
-			})
+			showError(t('spreed', 'Sending signaling message has failed.'))
```

Comment:
I intentionally removed the15 sec here.
The main reason is the message is not helping users. There is nothing they can do to "fix" this, like resending it etc. So unless the message is changed the default timeout seems good enough.